lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/ipv-core-main-journey.yaml

```diff
@@ -302,6 +311,35 @@ END:
     type: process
     lambda: build-client-oauth-response
 
+CRI_TICF:
+  response:
+    type: process
+    lambda: call-ticf-cri
+    lambdaInput:
+      journeyType: main
+  parent: CRI_STATE
+  events:
+    next:
+      targetState: IPV_SUCCESS_PAGE
+    fail-with-ci:
+      targetState: PYI_NO_MATCH
+
+CRI_TICF_ON_REUSE:
+  response:
+    type: process
+    lambda: call-ticf-cri
+    lambdaInput:
+      journeyType: reuse
+  parent: CRI_STATE
```

Comment:
Not sure of this parent ?

Reply:
Doesn't need one I don't think, but it should also handle an `error` event

Reply:
done